mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -169,6 +170,17 @@ CLASS net/minecraft/class_437 net/minecraft/client/gui/screen/Screen
 	METHOD method_40041 hide ([Lnet/minecraft/class_339;)V
 		ARG 0 widgets
 	METHOD method_41843 clearAndInit ()V
+	METHOD method_47412 setTooltips (Lnet/minecraft/class_7919;)V
+		ARG 1 tooltip
```

Comment:
Singular or plural?

Reply:
Enigma was rejecting singular so I assumed it was already in use

Reply:
@MattiDragon No, the param name

Reply:
Oh, yeah that's wrong

Reply:
> Enigma was rejecting singular so I assumed it was already in use

Yes, `MultiplayerScreen#setTooltip` already exists. This method and the field should probably be named based on the fact that the method must be called every rendering frame to differentiate these two.

Reply:
@enbrain 
> the method must be called every rendering frame
This is true for the generic tooltip system too